server/commands/userInviter.js

```diff
@@ -52,6 +52,8 @@ export default async function userInviter({
       name: invite.name,
       email: invite.email,
       service: null,
+      isAdmin: invite.role === "admin",
```

Comment:
We need to check the `inviteUser` policy in this api method to ensure only admins can invite.

Looks like this was an existing bug which was bad, but with the ability to create admins it would become a privilege escalation issue.